tests/deploy/sa-agent-pytest.yml

```diff
@@ -207,124 +207,152 @@ data:
         meta:
           package:
             name: system
-            version: 1.19.1
-      - id: 3f8cbb75-38e5-465e-a4aa-ee577d67eccd
-        revision: 2
-        name: sa-agent
+            version: 1.20.4
+      - id: 147702c3-409a-4a63-81a3-5d9ab84c620c
+        revision: 1
+        name: cloud_security_posture-2
         type: cloudbeat/cis_k8s
         data_stream:
           namespace: default
         use_output: default
-        package_policy_id: 3f8cbb75-38e5-465e-a4aa-ee577d67eccd
+        package_policy_id: 147702c3-409a-4a63-81a3-5d9ab84c620c
         streams:
           - id: >-
-              cloudbeat/cis_k8s-cloud_security_posture.findings-3f8cbb75-38e5-465e-a4aa-ee577d67eccd
+              cloudbeat/cis_k8s-cloud_security_posture.findings-147702c3-409a-4a63-81a3-5d9ab84c620c
             data_stream:
               type: logs
               dataset: cloud_security_posture.findings
             name: Findings
+            fetchers:
+              - name: kube-api
+              - name: process
+                directory: /hostfs
+                processes:
+                  etcd: null
+                  kube-apiserver: null
+                  kube-controller: null
+                  kube-scheduler: null
```

Comment:
I don't I see us using this format before, is it needed for a special reason?

Reply:
No. Current sa-agent yaml is a merge of newest yaml taken from cloud deployment 8.6 version and the current one.